### PR TITLE
Add SVG icons

### PR DESCRIPTION
**Pull request by spadgett**
Use SVG icons when available.

See https://github.com/openshift/origin-web-console/pull/2208
Requires https://github.com/openshift/origin-web-common/pull/213

screen shot 2017-10-03 at 3 03 29 pm

openshift web console 2017-10-03 15-57-53

### CODE REVIEW

**Comment by spadgett** [comment]
@jeff-phillips-18 PTAL

**Comment by jeff-phillips-18** [comment]
Looks good. One question, should there be methods to get the image for a given service instance thus allowing us to show the image in other places in the UI (like the service instance details page)?

**Comment by spadgett** [comment]
> One question, should there be methods to get the image for a given service instance thus allowing us to show the image in other places in the UI (like the service instance details page)?

Yeah, I know @ncameronbritt had started looking at showing icons generally on the overview. We could pretty easily add them for service instances since there is a clear icon, at least on the details page. @ncameronbritt Opinion?

File: src/components/create-from-builder/create-from-builder-info.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <div class="order-service-details">
   <div class="order-service-details-top">
     <div class="service-icon">
-      <span class="icon {{$ctrl.imageStream.iconClass}}" aria-hidden="true"></span>
+      <span ng-if="!$ctrl.imageStream.imageUrl" class="icon {{$ctrl.imageStreamiconClass}}" aria-hidden="true"></span>
+      <span ng-if="$ctrl.imageStream.imageUrl" class="image"><img ng-src="{{$ctrl.imageStream.imageUrl}}" alt=""></span>
```
[the remark this answers is not in the document]
Comment (spadgett) [reply src/components/create-from-builder/create-from-builder-info.html]: For `img` elements, I don't think `aria-hidden` is needed if `alt=""` is set.

https://www.w3.org/WAI/tutorials/images/decorative/

**Comment by spadgett** [comment]
@jeff-phillips-18 Opened #2221 to discuss using the icon elsewhere.

**Pull request merged**